images-list

```diff
@@ -248,15 +250,20 @@ k8s.gcr.io/prometheus-adapter/prometheus-adapter rancher/mirrored-prometheus-ada
 k8s.gcr.io/sig-storage/csi-attacher rancher/longhornio-csi-attacher v3.2.1
 k8s.gcr.io/sig-storage/csi-attacher rancher/mirrored-longhornio-csi-attacher v3.2.1
 k8s.gcr.io/sig-storage/csi-attacher rancher/mirrored-sig-storage-csi-attacher v3.2.0
+k8s.gcr.io/sig-storage/csi-attacher rancher/mirrored-sig-storage-csi-attacher v3.3.0
 k8s.gcr.io/sig-storage/csi-node-driver-registrar rancher/longhornio-csi-node-driver-registrar v2.3.0
 k8s.gcr.io/sig-storage/csi-node-driver-registrar rancher/mirrored-longhornio-csi-node-driver-registrar v2.3.0
+k8s.gcr.io/sig-storage/csi-node-driver-registrar rancher/mirrored-sig-storage-csi-node-driver-registrar v2.3.0
 k8s.gcr.io/sig-storage/csi-provisioner rancher/longhornio-csi-provisioner v2.1.2
 k8s.gcr.io/sig-storage/csi-provisioner rancher/mirrored-longhornio-csi-provisioner v2.1.2
 k8s.gcr.io/sig-storage/csi-provisioner rancher/mirrored-sig-storage-csi-provisioner v2.2.0
+k8s.gcr.io/sig-storage/csi-provisioner rancher/mirrored-sig-storage-csi-provisioner v3.0.0
 k8s.gcr.io/sig-storage/csi-resizer rancher/longhornio-csi-resizer v1.2.0
 k8s.gcr.io/sig-storage/csi-resizer rancher/mirrored-longhornio-csi-resizer v1.2.0
+k8s.gcr.io/sig-storage/csi-resizer rancher/mirrored-sig-storage-csi-resizer v1.3.0
 k8s.gcr.io/sig-storage/csi-snapshotter rancher/longhornio-csi-snapshotter v3.0.3
 k8s.gcr.io/sig-storage/csi-snapshotter rancher/mirrored-longhornio-csi-snapshotter v3.0.3
+k8s.gcr.io/sig-storage/livenessprobe rancher/mirrored-sig-storage-livenessprobe v2.4.0
```

Comment:
Was a repo for this image created? I don't see it in the EIO issue.

Reply:
Yes, here's the issue number is #730